src/bin.ts

```diff
@@ -138,7 +138,7 @@ if (args.length === 0) {
   process.exit(0);
 } else if (args.some((arg) => arg === "-v" || arg === "--version")) {
   const version = JSON.parse(
-    fs.readFileSync(path.join(__dirname, "..", "package.json"), "utf-8"),
+    fs.readFileSync(path.resolve(process.cwd(), "..", "package.json"), "utf-8"),
```

Comment:
This change here and a few lines later has the unexpected consequence that the `.download` folder is created in the user's project directory. Previously, it was created inside `node_modules` to not clutter the project directory. Can you revert his?